libcxx/test/libcxx/utilities/memory/ptr.align/assume_aligned.const_eval.verify.cpp

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is dual licensed under the MIT and the University of Illinois Open
```

Comment:
This looks like the wrong license to me.

Reply:
I had taken this from this file itself:

```
libcxx/test/libcxx/utilities/memory/ptr.align
assume_aligned.power2.verify.cpp
```

Reply:
Yes, that test also had an old license header. I've removed any remaining cases. Please update to the new header.